ci/run_tests.sh

```diff
@@ -23,6 +23,7 @@ if test "$CLANG_SANITIZER" != "TSAN" ; then
   # Additional threads are only created when the builtin UI starts, which
   # doesn't happen in the unit/functional tests
   run_test run_unittests
+  export NODE_PATH="$(npm root -g)"
```

Comment:
Doing this here is ugly, but I don't see a better way.

Reply:
I think plugin authors are expected to locally install the neovim npm package as a dependency in `package.json` to avoid global `require()`. The test cases I've created use the API directly instead of the relying on the host ([bin/cli.js](https://github.com/neovim/node-client/blob/master/bin/cli.js)) so `NODE_PATH` must be set.

I wonder if there's a way to break up the npm package into host-only (end users, global install) and plugin-only (plugin authors, local install) without source code duplication.

Reply:
I've been working on a branch that does this (host, plugin, "attach"/api), just have been really busy lately.